rocks/RocksDatabase.java

```diff
@@ -239,10 +236,25 @@ private void mayInitRocksLogger() {
         }
     }
 
-    private void validateEncoding(RocksStorage.Schema schemaStorage) {
-        int encoding = schemaStorage.getEncoding();
-        if (encoding != ENCODING_VERSION) {
-            throw TypeDBException.of(INCOMPATIBLE_ENCODING, name(), encoding, ENCODING_VERSION);
+    private void initialiseEncodingVersion() {
+        try {
+            rocksSchema.put(rocksSchemaPartitionMgr.get(Storage.Key.Partition.DEFAULT),
+                    ENCODING_VERSION_KEY.bytes().getBytes(), ByteArray.encodeInt(ENCODING_VERSION).getBytes());
+        } catch (RocksDBException e) {
+            throw TypeDBException.of(e);
+        }
+    }
```

Comment:
we do a direct PUT and skip all the locking mechanisms required in the previous version (`RocksStorage.java`) because we can be sure that no one is doing any concurrent operations on the storage during initialisation

Reply:
Makes sense. Can you also format the argument to `.put(...)` to each be on line 242, 243, and 244, with closing parentheses on line 245.